pkg/portal/graphql/app.go

```diff
@@ -198,11 +220,23 @@ var nodeApp = node(
 			},
 			"secretConfig": &graphql.Field{
 				Type: graphql.NewNonNull(secretConfig),
+				Args: graphql.FieldConfigArgument{
+					"unmaskedSecrets": &graphql.ArgumentConfig{
+						Type: graphql.NewNonNull(graphql.NewList(graphql.NewNonNull(secretKey))),
```

Comment:
Make the array nullable (keep the element non-null) so that existing queries are compatible.